DataFormats/GEMDigi/src/classes_def.xml

```diff
@@ -42,6 +42,42 @@
 <class name="MuonDigiCollection<GEMDetId,GEMCoPadDigi>"/>
 <class name="edm::Wrapper<MuonDigiCollection<GEMDetId,GEMCoPadDigi> >" splitLevel="0"/>
 
+<class name="GEMAMC13Status" ClassVersion="-1">
```

Comment:
Sorry, I miss the meaning of `ClassVersion="-1"`
It is the first time I see it: what is it meant to?

Reply:
@makortel 
Perhaps this should be 3 following cmssw conventions

Reply:
we have another example in DataFormats/ForwardDetId/src/classes_def.xml

Reply:
> @makortel
> Perhaps this should be 3 following cmssw conventions

Yes, I agree it should be 3. It would be good to fix it quickly in `master` and `CMSSW_12_0_X` as well (or does 12_0_0_pre5 already store these products?).

Reply:
@makortel 12_0_0_pre5 would store this already, but it is only stored and used for onlineDQM
Shall I do a PR now to fix this for both master and CMSSW_12_0_X?

Reply:
is this data really persisted? I thought that the status data is transient

Reply:
If it is transient then it would be best to mark the class as transient and remove the class version.
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#classes_def_xml_entries_for_a_tr

Reply:
Its use is mainly transient, but it would be useful to keep it like this so that we can analyze it in detail if needed.